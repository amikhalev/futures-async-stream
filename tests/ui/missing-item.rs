// compile-fail

#![deny(warnings)]
#![feature(async_await, generators)]

use futures_async_stream::async_stream;

#[async_stream] //~ ERROR unexpected end of input, expected `item`
async fn foo(a: String) {}

fn main() {}
