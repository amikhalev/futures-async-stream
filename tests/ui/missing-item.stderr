error: unexpected end of input, expected `item`
 --> $DIR/missing-item.rs:7:1
  |
7 | #[async_stream] //~ ERROR unexpected end of input, expected `item`
  | ^^^^^^^^^^^^^^^
